src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <form action="#"
+      method="post"
+      data-qa="big"
+      class="search_form"
+  >
```

Comment:
```suggestion
    <form 
        action="#"
        method="post"
        data-qa="big"
        class="search_form"
     >
```
[checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) p.5

Reply:
fix it all over your code